composer.lock

```diff
@@ -5045,16 +5169,16 @@
         },
         {
             "name": "twig/twig",
-            "version": "v2.6.2",
+            "version": "v2.7.2",
```

Comment:
:+1: Also waiting for [CVE-NONE-0001](https://symfony.com/blog/twig-sandbox-information-disclosure) fix to be merged to master.